src/main/java/uk/gov/digital/ho/hocs/search/application/RequestData.java

```diff
@@ -13,25 +13,13 @@
 @Component
 public class RequestData implements HandlerInterceptor {
 
-
     public static final String CORRELATION_ID_HEADER = "X-Correlation-Id";
     public static final String USER_ID_HEADER = "X-Auth-UserId";
     public static final String USERNAME_HEADER = "X-Auth-Username";
```

Comment:
I see a use for this, but I also see it as just another header as we have the `UserId`